pftp/tls.go

```diff
@@ -0,0 +1,180 @@
+package pftp
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"strings"
+)
+
+// TLS version codes
+const (
+	SSLv3  = 0x0300
+	TLSv1  = 0x0301
+	TLSv11 = 0x0302
+	TLSv12 = 0x0303
+)
+
+// get TLS protocol from string version name
+func getTLSProtocol(protocol string) uint16 {
+	switch protocol {
+	case "TLSv1":
+		return TLSv1
+	case "TLSv1.1":
+		return TLSv11
+	case "TLSv1.2":
+		return TLSv12
+	default:
+		return TLSv1 // the default TLS protocol is TLSv1.0
```

Comment:
I don't think that it is not better choice, because, TLSv1 has a lot of vulnerability

Reply:
I agree.

Use TLS v1.2 by default and make TLS v1.3 available.

Reply:
fixed!

https://github.com/pyama86/pftp/pull/38/commits/546079c420e26fece68b8933d0846e21eafd313e

Reply:
I just add log about connected TLS protocol version, for check TLS protocol usage rate.

```
time="2021-05-12T23:35:26+09:00" level=debug msg="[1] user:- addr:210.172.128.230:58356 TLS control connection finished with client. TLS protocol version: TLSv1.2"
```